File: client/ReciteWords/sdk/review/ReviewWordsDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace sdk.review.words
{
    /// <summary>
    /// 复习记录中的单词
    /// </summary>
    public class ReviewWordsDto
    {
        public long Id { get; set; }

        /// <summary>
        /// 复习reviewId
        /// </summary>
        public long ReviewId { get; set; }

        /// <summary>
        /// 单词wordId
        /// </summary>
        public long WordsId { get; set; }

        /// <summary>
        /// 拼写
        /// </summary>
        public string Spelling { get; set; }

        /// <summary>
        /// 发音
        /// </summary>
        public string Pronunciation { get; set; }

        /// <summary>
        /// 释义
        /// </summary>
        public string Explanation { get; set; }
    }
}

File: service/recite-words/src/main/java/org/noodle/bean/review/ReviewWordsDto.java
package org.noodle.bean.review;

/**
 * 复习记录中的单词
 * @author wanghy
 */
public class ReviewWordsDto {
    /**
     * 背诵单词记录id
     */
    private Long id;

    /**
     * 复习reviewId
     */
    private Long reviewId;

    /**
     * 单词wordId
     */
    private Long wordsId;

    /**
     * 拼写
     */
    private String spelling;

    /**
     * 发音
     */
    private String pronunciation;

    /**
     * 释义
     */
    private String explanation;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getReviewId() {
        return reviewId;
    }

    public void setReviewId(Long reviewId) {
        this.reviewId = reviewId;
    }

    public Long getWordsId() {
        return wordsId;
    }

    public void setWordsId(Long wordsId) {
        this.wordsId = wordsId;
    }

    public String getSpelling() {
        return spelling;
    }

    public void setSpelling(String spelling) {
        this.spelling = spelling;
    }

    public String getPronunciation() {
        return pronunciation;
    }

    public void setPronunciation(String pronunciation) {
        this.pronunciation = pronunciation;
    }

    public String getExplanation() {
        return explanation;
    }

    public void setExplanation(String explanation) {
        this.explanation = explanation;
    }
}
